contrib/mesos/pkg/scheduler/scheduler.go

```diff
@@ -923,6 +924,10 @@ func (ks *KubernetesScheduler) recoverTasks() error {
 		ks.slaves.checkAndAdd(slaveId, t.Offer.Host())
 	}
 	for _, pod := range podList.Items {
+		if _, isMirrorPod := pod.Annotations[kubelet.ConfigMirrorAnnotationKey]; isMirrorPod {
+			// mirrored pods are never reconciled
```

Comment:
I meant the comment. it doesn't add anything to what is immediate in the code line above. The "why" would be helpful.